.gitignore

```diff
@@ -5,3 +5,4 @@
 *~
 .project
 .pydevproject
+wsgi/media/**
```

Comment:
A trailing "/**" matches everything inside.
@yashLadha Sir, should I just add exact path `wsgi/media/photologue/photos/` as cache is formed inside that